src/main/scripts/festerize/README.md

```diff
@@ -0,0 +1,58 @@
+# festerize
+Script to upload CSV files to the UCLA Library Fester service.
+
+# Using festerize.py
+
+First, make sure you have Python 3 available (we recommend using [pyenv](https://github.com/pyenv/pyenv) if you need to 
+upgrade the version of python installed on your system)  and [install pipenv](https://pipenv.kennethreitz.org/en/latest/#install-pipenv-today). Then you can use pipenv to install the project's dependencies in a new virtual environment: 
+
+```
+pipenv install
+
+# or, if you intend to make changes to the festerize.py script, you can run
+
+pipenv install --editable .
+```
+> NOTE: if you're familiar with Python development and prefer not to use
+>pipenv, that's fine, do whatever is comfortable to you. This README will assume
+>that you are using pipenv.
+
+To run commands inside the new virtual environment, you can either enter `pipenv shell` to enter the virtual environment, or you can prefix your commands with `pipenv run`.
+
+You can then use the script to load one or more CSVs into Fester:
+
+```
+festerize --help
+Usage: festerize [OPTIONS] SRC...
+
+  FESTERIZE uploads CSV files to the UCLA Library Fester service.
+
+Options:
+  --server TEXT    URL the Fester service we are using. Default:
+                   https://iiif.library.ucla.edu
+  --endpoint TEXT  Service endpoint to use. Default: /collections
+  --out TEXT       Folder to store the results of festerizing. Default:
+                   ./output
+  --loglevel TEXT  Log level for Festerizer logs. Default: INFO, can also be
+                   DEBUG or ERROR
+  --help           Show this message and exit.
+```
+
+Note that the SRC argument above supports standard [filename globbing](https://en.wikipedia.org/wiki/Glob_(programming)
```

Comment:
Think you're missing a closing paren here? The output gets mangled: https://github.com/UCLALibrary/fester/tree/IIIF-574-csv-uploading/src/main/scripts/festerize